src/libraries/System.IO.Ports/Directory.Build.props

```diff
@@ -3,5 +3,6 @@
   <PropertyGroup>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
     <UnsupportedOSPlatforms>browser;ios;tvos</UnsupportedOSPlatforms>
+    <SupportedOSPlatforms>maccatalyst</SupportedOSPlatforms>
```

Comment:
Why do we need this line?

Reply:
Because MacCatalyst is a superset of iOS, see https://learn.microsoft.com/en-us/dotnet/standard/analyzers/platform-compat-analyzer#platform-inclusion

> [UnsupportedOSPlatform("iOS") implies that the API is not supported on iOS and MacCatalyst. You can use the [SupportedOSPlatform("MacCatalyst")] attribute to exclude this implied lack of support.